readme.md

```diff
@@ -105,34 +105,34 @@
 	- [Functional Programming](https://github.com/stoeffel/awesome-fp-js)
 	- [Observables](https://github.com/sindresorhus/awesome-observables)
 	- [npm scripts](https://github.com/RyanZim/awesome-npm-scripts) - Task runner.
-- [Swift](https://github.com/matteocrippa/awesome-swift)
+- [Swift](https://github.com/matteocrippa/awesome-swift)-  a general-purpose, multi-paradigm, compiled programming language developed by Apple Inc.
```

Comment:
Too generic of a description. Should mention its strengths and what it's mainly used for.

Reply:
Also, all descriptions should start with uppercase.